.drone.yml

```diff
@@ -45,6 +45,7 @@ steps:
 - name: lint-backend
   image: grafana/build-container:1.4.1
   commands:
+  - make gen-go
```

Comment:
Maybe not understanding sth - but why this is required to be called in CI? Looks like all things should be generated locally by a developer? Or at least if the `make gen-go` produces changes in git tree then this is a sign of a problem?